### ISSUE TITLE
error when starting server

### ISSUE THREAD

**Issue by joaofbfrade** [frida/frida#2478]
Frida:ERROR:../../../frida-core/src/linux/frida-helper-backend.vala:858:frida_inject_session_class_init: assertion failed: (local_libc != null)
Bail out! Frida:ERROR:../../../frida-core/src/linux/frida-helper-backend.vala:858:frida_inject_session_class_init: assertion failed: (local_libc != null)

Im getting this error, my device is rooted with magisk manager and i have lsposed

**Comment by rossSec** [comment]
Getting the same error here.

**Comment by chouex** [comment]
Getting the same error at `frida-server-16.0.12-android-arm64` and `frida-server-16.0.13-android-arm64` 
OS: Android 9
```
**
Frida:ERROR:../../../frida-core/src/linux/frida-helper-backend.vala:858:frida_inject_session_class_init: assertion failed: (local_libc != null)
Bail out! Frida:ERROR:../../../frida-core/src/linux/frida-helper-backend.vala:858:frida_inject_session_class_init: assertion failed: (local_libc != null)
Aborted
```

android 13 work fine

**Comment by dabiaotouxiang** [comment]
maybe you can try this

![1681660357320](https://user-images.githubusercontent.com/51227631/232324739-1e7f8941-35f4-42f8-b4d5-508973bda062.png)

**Comment by StormNkl** [comment]
> maybe you can try this
> 
> ![1681660357320](https://user-images.githubusercontent.com/51227631/232324739-1e7f8941-35f4-42f8-b4d5-508973bda062.png)

Hi , now it says : 
Failed to enumerate processes: unable to connect to remote frida-server: closed
my phone is xiaomi mi note 10 root with magisk + lsposed

**Comment by joaofbfrade** [comment]
i fixed it!  the guy on #2299 said to close Se linux
 
you need to use this command 

adb shell
setenforce 0

it worked for me, im using redmi note 8 pro and 16.011 frida server

![image](https://user-images.githubusercontent.com/22626546/232500721-4c0eb136-ff5b-441d-bebf-477e3b60d40c.png)

**Comment by joaofbfrade** [comment]
The command "adb shell setenforce 0" is used to temporarily disable SELinux (Security-Enhanced Linux) enforcement on an Android device that is connected to a computer via ADB (Android Debug Bridge).

SELinux is a security feature that provides a mechanism for enforcing mandatory access control policies on Android devices. It helps to protect the device from malicious activities by limiting the actions that system processes and applications can perform.
